designer/server/src/models/forms/editor-v2/pages.js

```diff
@@ -19,6 +19,14 @@ import {
 
 const BUTTON_SECONDARY_CLASS = 'govuk-button--secondary'
 
+/**
+ * @param {Page} page
+ */
+export function isGuidancePage(page) {
+  const components = hasComponents(page) ? page.components : []
+  return components.length >= 1 && components[0].type === ComponentType.Markdown
```

Comment:
Would this give a false positive for `{ page: { components: [{markdown component}, { formcomponent }] }}`?